sql-plugin/src/main/330+/scala/com/nvidia/spark/rapids/shims/Spark33XShims.scala

```diff
@@ -343,8 +343,8 @@ trait Spark33XShims extends Spark321PlusShims with Spark320PlusNonDBShims {
           override val childScans: scala.Seq[ScanMeta[_]] =
             Seq(GpuOverrides.wrapScan(p.scan, conf, Some(this)))
 
-          override def convertToGpu(): GpuExec =
-            GpuBatchScanExec(p.output, childScans.head.convertToGpu())
+          override def convertToGpu(): GpuExec = GpuBatchScanExec(p.output,
```

Comment:
so this doesn't look like for 33X shims we tag properly to not run on gpu when the runtimeFilters are present? I'm guessing when we override the Spark320PlusShims we dropped that accidentally 

```
 override def tagPlanForGpu(): Unit = {
            if (!p.runtimeFilters.isEmpty) {
              willNotWorkOnGpu("Runtime filtering (DPP) on datasource V2 is not supported")
            }
          }
```

Reply:
Nice catch!  I updated this to check for both runtime filtering and key grouped partitions.